docs/README.md

```diff
@@ -116,6 +116,18 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `optionsInputValueLabelContainer` | 
 | `optionsInputValueOptionBoxContainer` |
 | `textInputValue` |
+| `modalViewProductContainer` |
+| `modalViewProductImage` |
+| `modalViewProductName` |
+| `modalViewProductInfos` |
+| `modalViewDoneButton` |
+| `productAssemblyOptionItemCustomize` |
+| `productAssemblyOptionItemCustomize__label` |
+| `productAssemblyOptionItemImage` |
+| `productAssemblyOptionItemName` |
+| `productAssemblyGroupName` |
+| `productAssemblyGroupNameRow` |
+| `productAssemblyGroupRequiredTag` |
```

Comment:
They should be added to the table in alphabetical order! Could you update this part, please? :) Thank you!

Reply:
@guerreirobeatriz It's done! Could you continue the flow pls? 🙏